app/main.py

```diff
@@ -16,4 +16,26 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+# Creating dictionary where results will be added
```

Comment:
Please, remove the redundant comments. There should be no extra code in your PRs

Reply:
it's not redundant comment, it's comment I added for better understanding. Isn't it a good practice to add comments to your code?

Reply:
Your code is easy to understand. There's no need to explain it by adding additional comments

Reply:
Your code is easy to understand. There's no need to explain it by adding additional comments. You should add them when it's not

Reply:
noted, thanks